main.cpp

```diff
@@ -0,0 +1,228 @@
+// Configuration editor
+// Command line utiltiy to create configuration files
+// Valid arguments are init and edit
+
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <stdio.h>
+
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+
+const string CONFIG_FILE_PATH = "/users/bconway/SourceTree/School/CPT-180-27-Assignment-2-Config-Generator/app.cfg";
+
+
+int main(int argc, char *argv[]) {
+
+    string argument1 = "";
+    // Make sure that the argument isn't empty
+    if (argv[1] != nullptr) {
```

Comment:
@Arevyn24 the feature is actually already in the code. I had issues reading and writing the file at the same time. So I have the code creating a backup of the original file every time it enters the edit section of the code (I figured the backup file would be nice anyway) then I read the backup file line by line and if I don't need to edit it the line I write it to the output. If I need to edit the line (base on the argument or lack there of) I prompt the user to change the config. If the user hits enter without entering anything then I just write the original value, otherwise I write what they entered. This will allow the system to handle  config file of any length with out much re coding. I was also tempted to add comment lines and ignore those, but I didn't have the time. 

I also have it giving some instructions if there aren't any arguments. 

Feel free to look through the code for details. The backup file was the biggy though, because I was trying to read and write to the file at the same time and that just didn't work out.